docs/quick-start.md

```diff
@@ -0,0 +1,173 @@
+# Quick Start
+
+
+## Prepare your working directoy
+```bash
+# The name is only for demo, please use your own azure resources instead
+mkdir -p notation-akv-demo && cd notation-akv-demo
+AKV_NAME=yzhakv
+ACR_NAME=yzhregistry
+REGISTRY=${ACR_NAME}.azurecr.io
+REPO=net-monitor
+TAG=v1
+IMAGE=${REGISTRY_NAME}/${REPO}:${TAG}
+KEY_NAME=test16
+```
+
+## Create a CA
+Create a config file for openssl to create a valid CA
+```bash
+cat <<EOF > ./ca_ext.cnf
+[ v3_ca ]
+basicConstraints = CA:TRUE
+keyUsage = critical,keyCertSign
+extendedKeyUsage = codeSigning
+EOF
+```
+Sign the CA
+```bash
+# create a signing request to sign your root CA
+openssl req -new -newkey rsa:2048 -nodes -out ca.csr -keyout ca.key -extensions v3_ca
+
+# sign the root CA with ca_ext.cnf
+openssl x509 -signkey ca.key -days 365 -req -in ca.csr -set_serial 01 -out ca.crt -extensions v3_ca -extfile ./ca_ext.cnf
+```
```

Comment:
This can be simplified to a single line:
```
openssl req -x509 -sha256 -nodes -newkey rsa:2048 -keyout ca.key -out ca.crt -days 365 -subj "/CN=Test CA" -addext "keyUsage=critical,keyCertSign"
```
As a CA, we don't need `keyCertSign` and `codeSigning`.

Reply:
We don't need to set `id-kp-codeSigning` for CA certs

Reply:
Updated